airbyte-webapp/.storybook/main.ts

```diff
@@ -1,4 +1,7 @@
 module.exports = {
+  core: {
```

Comment:
:information_source: Required for making storybook run with webpack 5, which is used now in the updated `react-scripts` version.